src/main/resources/hudson/plugins/favorite/assets/css.jelly

```diff
@@ -0,0 +1,49 @@
+<?jelly escape-by-default='true'?>
```

Comment:
The favorite star is not vertically aligned

![dashboard jenkins 2016-11-22 08-16-33](https://cloud.githubusercontent.com/assets/50156/20501016/1057fe74-b08c-11e6-991e-e399a69941af.png)